### DISCUSSION TITLE
[Unknown Mod][Adventure map] Movement penalty for leaving a town/castle

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2606] [Ideas]
In the original game, the square at the entrance to the town/castle has the same movement penalty as the terrain under the town/castle entrance.

I always found it annoying that the movement penalty for leaving a castle is defined by the type of terrain under the castle entrance.
This penalty should always be 75%, just like the road penalty.

This is not an option.
This can be implemented directly in fheroes2.

**Comment by Branikolog** [comment]
@LeHerosInconnu 
I think, this shouldn't be even an option, but common operation. In all later games of HoMM series castle does not have penalty on entering. 
I bet, noone will notice, if we turn off penalty for this square.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> @LeHerosInconnu
> I think, this shouldn't be even an option, but common operation. In all later games of HoMM series castle does not have penalty on entering.
> I bet, noone will notice, if we turn off penalty for this square.

I agree with that. :)
The option was there for a possible purist. :D
